src/App/ImportUshahidiV2/Console/ImportUshahidiV2Command.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/**
+ * Ushahidi Import Console Commands
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\App\ImportUshahidiV2\Console;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Contracts\Bus\Dispatcher;
+use Ushahidi\App\ImportUshahidiV2;
+
+class ImportUshahidiV2Command extends Command
+{
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'import:ushahidiv2';
+
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'import:ushahidiv2
+                            {database : The name of the database to import}
+                            {--u|user : The username to connect to the DB}
+                            {--p|password : The password to connect to the DB}
+                            {--host : The database host to connect to}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Import Ushahidi V2 database';
+
+    protected $dispatcher;
+
+    public function __construct(Dispatcher $dispatcher)
+    {
+        parent::__construct();
+
+        $this->dispatcher = $dispatcher;
+    }
+
+    public function handle()
+    {
+        // Check we don't already have v3 data
```

Comment:
Is this a TODO, or is this a summary of what's happening below -- I'm assuming the former?

Reply:
A todo/me writing pseudo code then not bothering to implement it